nvflare/app_common/workflows/base_model_controller.py

```diff
@@ -129,8 +129,10 @@ def broadcast_model(
 
         if not isinstance(task_name, str):
             raise TypeError("task_name must be a string but got {}".format(type(task_name)))
-        if data and not isinstance(data, FLModel):
-            raise TypeError("data must be a FLModel or None but got {}".format(type(data)))
+        if not data:
```

Comment:
Why are we adding this check here? Could this break existing code?

Reply:
It does not make sense to broadcast_model() with an empty data. I checked all the places, we always create a FLModel object before calling this. This should not break existing codes.